### ISSUE TITLE
Self Assign to Add NVMEFix to improve battery life

### ISSUE THREAD

**Issue by aidanchandra** [aidanchandra/x1c7-hackintosh#6]
See title

**Comment by suhrmann** [comment]
Is this about the [acidanthera/NVMeFix kext](https://github.com/acidanthera/NVMeFix) ?

**Comment by aidanchandra** [comment]
@suhrmann Yes. There's some evidence to suggest this patch also fixes some previous hibernation issues seen and generally improves battery life.

**Comment by aidanchandra** [comment]
I've added it to my personal machine. I can't exactly test battery life well because my iStatMenus trial has expired.

I'll use for a bit and then add to master barring there are no issues.

**Comment by aidanchandra** [comment]
I haven't seen any issues with it thusfar. @suhrmann I'll make a branch. Would you mind also loading that EFI to your machine so we can get two tests going before merging to master?
